### PR TITLE
soc/da1469x: Fix CONFIG_SYS_CLOCK_*_PER_SEC defines

### PR DESCRIPTION
**Pull request by ydamigos**
Needs PR #73403 for function "dt_node_ph_prop_path"

The function is used so the following configuration options for DA1469x:

SYS_CLOCK_HW_CYCLES_PER_SEC
SYS_CLOCK_TICKS_PER_SEC

could get their values according to lp_clock node's clock-src property.

### CODE REVIEW

**Comment by ydamigos** [comment]
"Compliance Checks" failure seems unrelated to PR changes.

File: scripts/kconfig/kconfigfunctions.py
Comment (andrzej-kaczmarek) [inline scripts/kconfig/kconfigfunctions.py:1]: this should be a separate PR

Comment (ydamigos) [reply scripts/kconfig/kconfigfunctions.py:1]: Opened PR #73403

**Review by ioannis-karachalios — APPROVED** [review]

**Comment by ydamigos** [comment]
PR #73403 was merged. Removing `DNM` label.

**Review by blauret — APPROVED** [review]
